src/Kerbalism/UI/Monitor.cs

```diff
@@ -610,10 +610,8 @@ void Indicator_signal(Panel p, Vessel v, Vessel_info vi)
 			ConnectionInfo conn = vi.connection;
 			bool remotetech = RemoteTech.Enabled;
 
```

Comment:
```// signal strength or when using RemoteTech signal delay 
    string signal_str = remotetech ?
        !conn.linked ? "----" : conn.strength > Double.Epsilon ? KSPUtil.dateTimeFormatter.PrintTimeStampCompact(conn.strength) : Localizer.Format("#KERBALISM_Generic_NONE") :
         Lib.HumanReadablePerc(conn.strength, "F2"); ```

Needs to have its identifier changed to something like `string delay_str =` so RemoteTech can still show the Signal delay, or has it been ditched in favour of RT's display in the top left ?

Reply:
In Discord, I asked about it and only Gordon said that the "Top left is enough", then I removed it.